### PR TITLE
Create YashasSingh.md

### PR DESCRIPTION
**Pull request by YashasSingh**
## Describe your project in 1-3 sentences
I am building 2 projects, the first project is an internet-wide ad blocker and the second project is a virtual assistant hologram alarm clock. The virtual assistant hologram alarm clock is a combination of multiple projects made by me previously along with adding hardware to the raspberry pi

## Hack Club Slack handle
@Checkers

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

File: YashasSingh.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+ This project also features an alarm clock which will play music until the user is awake and out of the room.
+ 
+ 
+ ## Plan
+ 
+ Set up rasberry pi
+ Connect the screen, camera, audio hat module
+ Install a voice reconigition software to convert wavelengths into text
+ Create a word that starts the program ex. Alexa or Ok Google
+ Use Open Ai that runs when start word is said (Alexa, Ok Goolgle)
```
Comment (exu3) [inline YashasSingh.md:47]: What do you mean by "Open Ai"? Is this an open source AI model? Sorry, I'm not familiar.

File: YashasSingh.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+ Say the response from the speaker
```
Comment (exu3) [inline YashasSingh.md:48]: What speaker will you be using?

File: YashasSingh.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+ The clock software can be dowloaded from other pre-made clock sofware on git hub
```
Comment (exu3) [inline YashasSingh.md:49]: Can you link an example?

File: YashasSingh.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+ For example if the user gives a face of pain then the assitant can tell the user what do. 
+ The hologram feature will show a random pictures like jellyfish floating when the virtual assitant is not in use.
+ This project also features an alarm clock which will play music until the user is awake and out of the room.
+ 
+ 
+ ## Plan
+ 
+ Set up rasberry pi
+ Connect the screen, camera, audio hat module
+ Install a voice reconigition software to convert wavelengths into text
```
Comment (exu3) [inline YashasSingh.md:45]: Can you link an example?

**Review by exu3 — CHANGES_REQUESTED** [review]
Unfortunately, we can't approve the RPI Ad Blocker project as it does not meet the requirements of the project being technically challenging. It seems very basic in nature as you are just configuring open source software on a Raspberry Pi Zero.

The second project, however, looks like it would be an interesting project. I left some comments on parts of your proposal I found unclear. Let me know if you have any questions!

File: YashasSingh.md (hunk 1 above)
Comment (YashasSingh) [reply YashasSingh.md:47]: Yes this is an open source AI model that has already pre- trained
 import os
import openai

openai.api_key = os.getenv("OPENAI_API_KEY")

response = openai.Completion.create(
  model="text-davinci-003",
09  prompt="What day is it today"  temperature=0.6,=-
  max_tokens=150,
  top_p=1,
  frequency_penalty=1,
  presence_penalty=1
)

Comment (YashasSingh) [reply YashasSingh.md:47]: above is an example of the code^

File: YashasSingh.md (hunk 3 above)
Comment (YashasSingh) [reply YashasSingh.md:26]: sure here is a clock made by someone else I will be doing something similar.

**Comment by YashasSingh** [comment]
@exu3 is this okay

**Comment by YashasSingh** [comment]
@exu3 Is this good

**Comment by YashasSingh** [comment]
Hey, @LucasHT22 I do realize that you help people from the sprig however it seems I am not receiving replies from @exu3 is there any way I can contact that person?

**Comment by LucasHT22** [comment]
You can contact through Slack! But be patient, the demand on /winter is quite high:)

File: YashasSingh.md (hunk 3 above)
Comment (exu3) [reply YashasSingh.md:26]: I don't see a link. Did you add it to you markdown file?

File: YashasSingh.md (hunk 4 above)
Comment (exu3) [reply YashasSingh.md:22]: Can you add this to your markdown document?

Comment (YashasSingh) [reply YashasSingh.md]: Okay sure

File: YashasSingh.md (hunk 3 above)
Comment (YashasSingh) [reply YashasSingh.md]: it seems the link did not send here it is also I have added it on the markdown https://opensource.com/article/17/7/raspberry-pi-clock

**Review by sampoder — APPROVED** [review]
Looks like this is good to go! @YashasSingh enjoy a Winter Hardware Wonderland ❄️!

**Review by MelSmith104 — COMMENTED** [review]
hey @YashasSingh! this project looks so cool! it sounds like it would be very useful to have in the house :) im dm'ing you right now to chat about next steps!

**Comment by YashasSingh** [comment]
Thank you @sampoder 
@MelSmith104 I have replied on slack

**Review by MelSmith104 — APPROVED** [review]

**Pull request merged**